### PR TITLE
Add multiline option to Pretty.ColorizeJSON

### PR DESCRIPTION
**Pull request by NicolasMahe**
Very simple PR to add the multiline option to Pretty.ColorizeJSON function.
It is useful for a following PR.

### CODE REVIEW

File: utils/pretty/pretty.go [hunk 1]
```diff
@@ -209,14 +209,16 @@ func (p *Pretty) Colorize(c *color.Color, msg string) string {
...
-func (p *Pretty) ColorizeJSON(keyColor *color.Color, valueColor *color.Color, data []byte) []byte {
+func (p *Pretty) ColorizeJSON(keyColor *color.Color, valueColor *color.Color, multiline bool, data []byte) []byte {
 	if p.noColor {
 		return data
 	}
 
 	f := prettyjson.NewFormatter()
-	f.Indent = 0
-	f.Newline = ""
+	if multiline == false {
```
Comment (krhubert) [inline utils/pretty/pretty.go]: !multiline

Comment (NicolasMahe) [reply utils/pretty/pretty.go]: Updated

Comment (krhubert) [reply utils/pretty/pretty.go]: It's go way:

```
➜  a git:(dev) ✗ gometalinter .
main.go:7:5:warning: should omit comparison to bool constant, can be simplified to !a (S1002) (megacheck)
```

Also go source code:

```
➜  src gogrep 'if !' |wc -l 
8661
➜  src gogrep 'if.*== false' |wc -l
17
➜  src 
```

Comment (antho1404) [reply utils/pretty/pretty.go]: I don't like the `false == multiline` it's more "conventional but we never do that anywhere in the code. I think it's better to have consistency. Or if you want just change the parameter with `inline` like that you have a `if inline` ;)

Comment (NicolasMahe) [reply utils/pretty/pretty.go]: Ok guys, I give up on this ;)

**Pull request merged**